packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -139,8 +139,8 @@ abstract class FlutterCommand extends Command<void> {
   /// Will be `null` until the top-most command has begun execution.
   static FlutterCommand? get current => context.get<FlutterCommand>();
 
-  /// The option name for a custom observatory port.
-  static const String observatoryPortOption = 'observatory-port';
```

Comment:
To prevent this being a breaking change to Google, can you maintain both the observatory-port and vm-service-port for the short term?

Reply:
We add `observatory-port` as an alias below. Should that not be sufficient?